util/travis

```diff
@@ -38,7 +38,7 @@ function linux {
   HBA_CONF="$PG_DIR"/pg_hba.conf
   CONF="$PG_DIR"/postgresql.conf
 
-  sudo -E env PATH="$PATH" LD_LIBRARY_PATH="$LD_LIBRARY_PATH" make install
+  sudo -E env PATH="$PATH" LD_LIBRARY_PATH="$LD_LIBRARY_PATH" (cd jsoncdc; make install)
```

Comment:
I am starting to think this is a big enough experiment that a repo-level separation might be good. The `PostgresRSPetStore` is a challenging project and we might have to try a bunch of different ways to get a story for modularity that we like.